cudamapper/src/cudamapper_utils.cpp

```diff
@@ -65,5 +66,63 @@ void fuse_overlaps(std::vector<Overlap>& fused_overlaps, const std::vector<Overl
     set_relative_strand(fused_overlap);
     fused_overlaps.push_back(fused_overlap);
 }
+
+std::string string_slice(const std::string& s, std::size_t start, std::size_t end)
+{
+    return s.substr(start, end - start);
+}
+
+std::vector<std::string> kmerize_string(const std::string& s, int kmer_length, int stride)
```

Comment:
Again, you could use `string_view` in order to avoid dynamic memory allocations and copies for all these strings

Reply:
Addressed in previous comment and should be prioritized in the GPU port.